blivet/static_data/lvm_info.py

```diff
@@ -56,8 +62,14 @@ class PVsInfo(object):
     @property
     def cache(self):
         if self._pvs_cache is None:
-            pvs = blockdev.lvm.pvs()
             self._pvs_cache = dict()  # pylint: disable=attribute-defined-outside-init
+
+            try:
+                pvs = blockdev.lvm.pvs()
+            except blockdev.BlockDevNotImplementedError:  # pylint: disable=catching-non-exception
+                log.error("libblockdev lvm plugin is missing")
+                return self._pvs_cache
```

Comment:
I see it now. I got confused by two different ways the same is achieved for PVs and LVs.